### ISSUE TITLE
comments from Rob Wilton: rc:yang-data vs RFC8791

### ISSUE THREAD

**Issue by mcr** [core-wg/yang-cbor#28]
~~~~
(3) This draft makes use of the rc:yang-data extension.  
Was there any discussion about using "YANG Data Structure Extensions" (RFC 8791) instead, which is meant to be a cleaner formulation of 
the rc:yang-data extension, and without the dependency on RESTCONF?  
I would suggest that using RFC 8791 would be preferable if possible.
~~~~

**Comment by abierman** [comment]
This is a duplicate of #48 I think

**Comment by cabo** [comment]
We don't have to follow the structure and weird encoding scheme of RFC 7952.
(Or more strongly, we can't, as there is no way to add a @ to a SID :-)
We probably should define a tag that takes metadata and data, in that order (so it needs to contain, say, a two-element array).
Is that something we want to do now?

**Comment by cabo** [comment]
Was closed by #116
